src/Microsoft.Tye.Hosting/DockerImagePuller.cs

```diff
@@ -66,16 +64,39 @@ private async Task PullContainerAsync(string image)
         {
             await Task.Yield();
 
+            string name = image;
+            string version = "latest";
+            var idx = image.LastIndexOf(":");
```

Comment:
Do you need to do this? Doesn't reference handle it?

Reply:
No, also fixed. I just mixed the previous approach, it is not longer required.